handbook/business-operations/ceo-handbook.md

```diff
@@ -111,17 +111,30 @@ Every month Charlie will do the prep work for the monthly "✌️ All hands 🖐
 
 The day before the All Hands, Mike will prepare slides that reflect the CEO vision and focus. 
 
+### Post all hands
+
+The EA / CEO will post the link to the All hands meeting in #general slack channel
+Template to use:
+Thanks to everyone who contributed to today's "All hands" call.
+
+:tv: If you weren't able to attend, please **watch the recording** _(1.5x playback supported)_.
+
+You can also grab a copy of the [original slides](https://fleetdm.com/handbook/business-operations#all-hands)
+ 
+ Include instructions for decorating that template with the link to the Gong recording, including a generic screenshot demonstrating how to generate it
+ Include example screenshot of how the Slack post is supposed to look
+ include instructions about attaching a PDF of slides
```

Comment:
Instead of the three lines above about what to include in this PR, we can include those things in this PR.  The steps are:

1. Open Gong recording and click "Share call"
2. Click "Share with customers"
3. Then copy the link and light up "Watch the recording" text with that link.

Reply:
To upload the pdf, in Slack, attach the PDF as part of the post to the #general channel.

Reply:
For the example screenshot use the tactic above of including a screenshot, but use the other screenshot from the issue, so there's an example of how it's supposed to look.

Reply:
@Sampfluger88